perf/lmbench.py

```diff
@@ -101,6 +102,8 @@ def setUp(self):
                        (fsdir, config_file, tmp_config_file))
         process.system("sed 's!^FILE=.*$!FILE=%s!' '%s'  '%s'" %
                        (temp_file, tmp_config_file, config_file))
+        process.system("sed 's!^MB=.*$!MB=%s!' '%s'  '%s'" %
```

Comment:
As this repeats for some params, can we have a small `replace` function to do the same?

Reply:
Better if that can be targeted towards avocado utility too, IMO, would be very useful. @narasimhan-v Any suggestions on this.

Reply:
1. This is better if done natively in python. Something like below:
```
with open(config_file, "r") as in_file:
    with open(tmp_config_file, "w") as out_file:
        for line in in_file:
            out_file.write(line.replace('old', 'new'))
```

2. Whenever some code is repeating, better to have a loop / function. If that serves multiple tests, then utility. I think local function would do here.

Reply:
@nasastry The first change is from `config_file` to `tmp_config_file`. The other two seems to be opposite of it. Is this right?

Reply:
@harish-24 
`sed` tool changes they are capturing in to a temporary file and then replace original file with the temporary one. This is common for `sed` related operation.
@narasimhan-v and @harish-24 
Let me try a way to replace `sed` commands with `python` code. Will let you know asap.